src/commands/rails/index.ts

```diff
@@ -5,6 +5,23 @@ import { Gnarrc } from '../../utils/gnarrc'
 export class Rails {
   static command = 'rails'
 
+  static async init(name: string, extraArgs: string) {
+    process.stdout.write(`Setting up Rails New with Gnarly opinions...\n`)
```

Comment:
We print some crazy gnar beast ASCII art to the terminal 😏

Reply:
I had only low-quality gnar beast images, but I think this can be added to the GNIP board as an easy first project!